### ISSUE TITLE
Update examples using occupancy dataset with sklearn.model_selection.TimeSeriesSplit

### ISSUE THREAD

**Issue by Kautumn06** [DistrictDataLabs/yellowbrick#543]
In our documentation, several of the examples use the `occupancy` dataset, which contains time series data. To split the data into the training and test sets, they use `sklearn.model_selection.train_test_split`, which assumes the observations are independent and shuffles the data points by default. However, this wouldn't be true for time series data such as `occupancy` (which includes per-minute observations from February 4, 2015 through February 18, 2015). So instead, we should use `sklearn.model_selection.TimeSeriesSplit`. Here is a short excerpt from its [documentation](http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.TimeSeriesSplit.html):

> Time Series cross-validator
> 
> Provides train/test indices to split time series data samples that are observed at fixed time intervals, in train/test sets. In each split, test indices must be higher than before, and thus shuffling in cross validator is inappropriate.

@DistrictDataLabs/team-oz-maintainers

**Comment by lwgray** [comment]
@Kautumn06 great catch!
